twitter-clone/app/src/main/java/io/github/vnicius/twitterclone/ui/common/adapters/TweetsAdapter.kt

```diff
@@ -38,12 +39,8 @@ class TweetsAdapter(
         )
     }
 
-    override fun getItemCount(): Int {
-        return tweets.size
-    }
-
     override fun onBindViewHolder(vh: ViewHolder, index: Int) {
-        vh.bindView(tweets[index])
+        vh.bindView(getItem(index)!!)
```

Comment:
Never _ever_ **ever** `never` ***never*** ever use force unwrap. Remove the `!!` !!!!!